project/plugins.sbt

```diff
@@ -1,3 +1,4 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.3")
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
+addSbtPlugin("io.shiftleft" % "sbt-ci-release-early" % "2.0.32")
```

Comment:
Not related to this PR. I wonder if there is another plugin/CI-flow that auto-bumps the SBT versions, scala versions, etc (and if that is really needed).